sandbox/settings.py

```diff
@@ -270,33 +268,30 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         {
             # We configure default values that work with the test configuration of
             # github.com/openfun/openedx-docker.
+            "BASE_URL": values.Value(environ_name="EDX_BASE_URL", environ_prefix=None),
+            # Django backend
             "BACKEND": values.Value(
-                "richie.apps.courses.lms.base.BaseLMSBackend",
+                "richie.apps.courses.lms.edx.EdXLMSBackend",
                 environ_name="EDX_BACKEND",
                 environ_prefix=None,
             ),
-            "JS_BACKEND": values.Value(
-                "base",
-                environ_name="EDX_JS_BACKEND",
-                environ_prefix=None,
-            ),
             "COURSE_REGEX": values.Value(
-                r".*", environ_name="EDX_COURSE_REGEX", environ_prefix=None
-            ),
-            "SELECTOR_REGEX": values.Value(
-                r"^(?P<course_id>.*)$",
-                environ_name="EDX_SELECTOR_REGEX",
+                r"^.*/courses/(?P<course_id>.*)/course/?$",
```

Comment:
my understanding was that /info was in dogwood and that the new versions are using /course. So I thought of setting it to /info only for dogwood...

Reply:
Yes /info was in dogwood and newest versions are using /course. But newest versions are also including a redirect from `/info` to `/course`, so it is simplier to use /info.

Reply:
Since we are the only ones on earth to still run dogwood/eucalyptus I think we should go for the future proof path and do what we have to do on our dogwood/eucalyptus sites...

Reply:
Ah ah ok !